queue/QueueRunner.feature

```diff
@@ -1,13 +1,12 @@
 Feature: Command and control using a message broker
 
   Background:
-    Given I start with a clean broker having a request and a reponse queue
-    And a client that connects to the queues
+    Given I start with a clean broker for user "testuser" having a request and a response queue and a client that connects to the queues
```

Comment:
The username will no longer be included in the Spec, it's name should be auto-generated.

It is better to have them on two lines as they are separate concerns. One is a queue, the other is a client.

Reply:
I have pushed tag  v0.24. Please use that one.